spec/features/user_post_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'user/posts', type: :system do
+  before(:each) do
+    driven_by(:rack_test)
+    @user = User.first
+    @post = @user.posts.first
+    @comment = @post.comments.first
+  end
+
+  it 'has username' do
+    visit user_posts_path(@user)
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'has profile picture' do
+    visit user_posts_path(@user)
+    expect(page).to have_css('img')
+  end
+
+  it 'has number of posts' do
+    visit user_posts_path(@user)
+    expect(page).to have_content("Number of Posts: #{@user.posts_count}")
+  end
+
+  it 'has post title' do
+    visit user_posts_path(@user)
+    expect(page).to have_content(@post.title)
+  end
+
+  it 'has post body' do
+    visit user_posts_path(@user)
+    expect(page).to have_content(@post.text)
+  end
+
+  it 'has comments' do
+    visit user_posts_path(@user)
+    expect(page).to have_content(@comment.text)
+  end
+
+  it 'has comment count' do
+    visit user_posts_path(@user)
+    expect(page).to have_content("Comments: #{@post.comments_count}")
+  end
+
+  it 'has like count' do
+    visit user_posts_path(@user)
+    expect(page).to have_content("Likes: #{@post.likes_count}")
```

Comment:
Please add the following tests because you forgot to add them:
- [ ] a test that asserts you can see a section for pagination if there are more posts than fit on the view as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20a%20section%20for%20pagination%20if%20there%20are%20more%20posts%20than%20fit%20on%20the%20view.).
- [ ] a test that redirects to the post's show page when you click on a post. as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20on%20a%20post%2C%20it%20redirects%20me%20to%20that%20post%27s%20show%20page.).